### PR TITLE
Features/sbau 4130

### PR DESCRIPTION
**Pull request by rntsferreira**

### CODE REVIEW

**Comment by rntsferreira** [comment]
- Move bootstrap gulp tasks into the root gulp tasks
- Remove the SASS bower dependency and find a suitable NPM replacement

@danciu FYI

File: gulpfile.js [hunk 1]
```diff
@@ -1,8 +1,11 @@
-var  gulp = require('gulp'),
-  gutil = require('gulp-util'),
-  concat = require('gulp-concat'),
-  rename = require('gulp-rename'),
-  sass = require('gulp-sass');
+var gulp = require('gulp'),
+    sass = require('gulp-sass');
+    clean = require('gulp-clean'),
+    gutil = require('gulp-util'),
```
Comment (azz0r) [inline gulpfile.js]: is gutil actually used? (be better named as "util" also)

**Comment by azz0r** [comment]
:+1: 

Not sure about the gutil thing but otherwise signed off from me.

@danciu has the honour of merging :)

**Comment by rntsferreira** [comment]
@danciu 💤 😴

**Comment by azz0r** [comment]
Well remembered, merged!

**Pull request merged**